packages/core/src/connectors/aliyun-dm/send-single-mail.ts

```diff
@@ -0,0 +1,35 @@
+import { PublicParameters, request } from '../utilities/aliyun';
+
+/**
+ * @doc https://help.aliyun.com/document_detail/29444.html
+ *
+ */
+interface SingleSendMail {
+  AccountName: string;
+  AddressType: '0' | '1';
+  ReplyToAddress: 'true' | 'false';
+  Subject: string;
+  ToAddress: string;
+  ClickTrace?: '0' | '1';
+  FromAlias?: string;
+  HtmlBody?: string;
+  TagName?: string;
+  TextBody?: string;
+}
+
+const Endpoint = 'https://dm.aliyuncs.com/';
+
+/**
+ * @doc https://help.aliyun.com/document_detail/29444.html
+ *
+ */
+export const sendSingleMail = async (
+  parameters: PublicParameters & SingleSendMail,
+  accessKeySecret: string
```

Comment:
`sendSingleMail` & `SingleSendMail` are confusing. would you like to align to usage?

Reply:
this naming convention comes from aliyun DM docs lol

Reply:
It sucks, but still aligned with `singleSendMail`